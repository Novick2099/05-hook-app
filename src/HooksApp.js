import React from 'react'

export const HooksApp = () => {
    return (
        <div>
            <p>Hola Mundo</p>
        </div>
    )
}
