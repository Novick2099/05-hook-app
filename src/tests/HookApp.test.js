import React from 'react';
import {HooksApp} from "../HooksApp";
import {mount} from "enzyme";



describe('HookApp',()=>{
  it('Should match with snapShot', ()=>{
    const component = mount(<HooksApp/>);
    expect(component).toMatchSnapshot();
    console.log(component.html());
  });
});
