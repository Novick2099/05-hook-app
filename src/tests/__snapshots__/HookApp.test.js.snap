// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp Should match with snapShot 1`] = `
<div>
  <p>
    Hola Mundo
  </p>
</div>
`;
